### ISSUE TITLE
Improve Height calculation 

### ISSUE THREAD

**Issue by Drachenkaetzchen** [gregdavill/KiBuzzard#121]
It seems the height is taken from the bounding box, which ends up in odd heights. Here's an example:

These are the settings I used:

![image](https://github.com/gregdavill/KiBuzzard/assets/162974/96f2e2d6-9250-407c-9537-32b4ef8975fa)

Then I put 2 labels next to each other, one with the text `max 2A` and one with the text `max 2Aj`. As you can see, both the `max 2Aj` label's width and height are smaller.

![image](https://github.com/gregdavill/KiBuzzard/assets/162974/7ab60c17-61fe-4b4d-9eff-b3d14f71ae65)

**Comment by gregdavill** [comment]
Yes this is a known issue, #115 We've not rolled the fix into a release just yet, but a fix is available.
Can you try the latest commit? 6d79f60b 

Now the text `2A` will be scaled to 1.3mm in height. I'm not sure we have a good solution to handle characters below the baseline yet.
![image](https://github.com/gregdavill/KiBuzzard/assets/344310/d9de12b0-33ec-4f8d-b61d-d99895021e2b)

**Comment by Drachenkaetzchen** [comment]
Thank you, I missed that issue!

I'll try it out tomorrow or in the next few days - can we keep this issue open so I don't forget to check?

Regarding the baseline issue, I understand that's tricky. How about setting an overall height of the label? I think there should be two items: "define height" and "set font size". If I set the height to, let's say, 1.3mm, and choose a font size which will exceed the 1.3mm height when I use "j", I know that I should probably reduce the font size.
